pandas/core/ops/__init__.py

```diff
@@ -522,6 +522,16 @@ def _combine_series_frame(left, right, func, axis: int, str_rep: str):
         new_data = dispatch_to_series(left, right, func)
 
     else:
+        rvalues = right._values
```

Comment:
can you put this in an elif (and leave out 533, i believe you can just set new_data=bm, then leave line 535 as the else

Reply:
ATM this is mirroring the code in the axis==0 case above; in a follow-up we'll de-duplicate these two chunks